src/ServiceControl.Audit.UnitTests/ApprovalFiles/APIApprovals.CustomCheckDetails.approved.txt

```diff
@@ -1,3 +1,4 @@
 ServiceControl Health: Audit Message Ingestion Process
 ServiceControl.Audit Health: Audit Message Ingestion
+Storage: CheckRavenDBIndexLag
```

Comment:
@tmasternak @SzymonPobiega do we have any naming guidelines for these?

Reply:
I don't believe so. They should be human readable.

Reply:
How about

Storage index: Number of audit documents behind

Reply:
and now I wonder why we don't put them all into ServiceControl/.Audit Health category